src/main/webapp/WEB-INF/static/js/trialmanager/germplasm-study-source/germplasmStudySource.js

```diff
@@ -1,7 +1,7 @@
 (function () {
 	'use strict';
 
-	var germplasmStudySourceModule = angular.module('germplasm-study-source', []);
+	var germplasmStudySourceModule = angular.module('germplasm-study-source', [])
```

Comment:
Did you really mean to remove the semicolon here?

Reply:
@darla-leafnode , 
Apologies, I think I removed it by mistake, I reverted changes